gradle/verification-metadata.xml

```diff
@@ -866,9 +866,9 @@
             <pgp value="7186BBF993566D8C2F4F7ED7D945E643368FEF62"/>
          </artifact>
       </component>
-      <component group="io.github.gradle" name="gradle-enterprise-conventions-plugin" version="0.7.6">
-         <artifact name="gradle-enterprise-conventions-plugin-0.7.6.jar">
-            <sha256 value="b3ff4bba6c5ac6e50e7225c7d1f69c0a9bc2ae9f3ce2bfacdc3f1b3999c2e758" origin="Verified" reason="Artifact is not signed"/>
+      <component group="io.github.gradle" name="gradle-enterprise-conventions-plugin" version="0.8.0">
+         <artifact name="gradle-enterprise-conventions-plugin-0.8.0.jar">
+            <sha256 value="9fb7ce8df50492a462b34f4f009a115932780aaa80658eb6209569827f141042" origin="Generated by Gradle"/>
```

Comment:
❌ "Generated by Gradle" needs to be updated, see https://github.com/gradle/gradle/blob/a9347fc8ce82f4e958834de1fad17a458ea6b656/gradle/verification-metadata.xml#L38